test-src/org/pentaho/reporting/platform/plugin/async/AsyncIT.java

```diff
@@ -767,44 +767,6 @@ public void testScheduleNoSession() throws Exception {
     assertTrue( flag[ 0 ] );
   }
 
-  @Test
```

Comment:
Yeah, its so nice to have randomized test results. Keeps everyone on their toes. :)

This test wont be missed as it is more busy testing the framework than testing the actual code, even though the statistics now show a gaping hole here. Can you make these validations a plain unit-test (ie test direct invocations of the JobManager) to see that recalculate and schedule have been called correctly?